uPortal-security/uPortal-security-mvc/src/test/java/org/apereo/portal/url/CasLoginRefUrlEncoderTest.java

```diff
@@ -16,12 +16,15 @@
 
 import static org.junit.Assert.assertEquals;
 
+import com.google.common.collect.Sets;
 import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
 import org.junit.Before;
 import org.junit.Test;
 import org.springframework.mock.web.MockHttpServletRequest;
 
-/** @Author James Wennmacher, jwennmacher@unicon.net */
+/** */
```

Comment:
:art: empty doc comment?

Reply:
@Author is not requested it seems, so what do you want that I add

Reply:
Or remove the comment entirely

Reply:
I pushed a change and added a comment, but several other test class have a such empty comment, that's why I did it...